### DISCUSSION TITLE
No way to get dividers back to old color

### DISCUSSION THREAD

**Discussion by sveltarne** [saadeghi/daisyui#2532] [Ideas / feature requests]
in v4, daisyUI added new divider colors, and made the default color base-300. However, there's no way to get it back to the old color, which seems to be base-200 (or whatever text is)? anyway, while neutral looks similar to the old color on the website with dark theme, not every theme has a light colored neutral.

**Comment by saadeghi** [comment]
You're right, the color contrast is not good on dark themes.  
I will change it back to the old color in version 4.0.9 today.

**Reply by sveltarne** [reply]
ok, thanks.
